prevodach.js

```diff
@@ -0,0 +1,48 @@
+const axios = require('axios').default;
+const { v4: uuidv4 } = require('uuid');
+
+// PLEASE DONT STEAL THIS
+// GCP SECRET VALUE ?
+const { translatorKey } = require('config.json');
+var endpoint = 'https://api.cognitive.microsofttranslator.com';
+
+module.exports.swedishToEnglish = async (message) => {
+  const translation = await translate(message, ['en']);
+
+  // return only english FOR NOW
+  return translation[0].translations.find((t) => t.to == 'en');
+};
+
+async function translate(message, targetLanguages) {
+  const request = {
+    baseURL: endpoint,
+    url: '/translate',
+    method: 'post',
+    headers: {
+      'Ocp-Apim-Subscription-Key': translatorKey,
+      // Add your location, also known as region. The default is global.
+      // This is required if using a Cognitive Services resource.
+      'Ocp-Apim-Subscription-Region': 'westeurope',
+      'Content-type': 'application/json',
+      'X-ClientTraceId': uuidv4().toString(),
+    },
+    params: {
+      'api-version': '3.0',
+      'to': targetLanguages,
```

Comment:
'to': targetLanguages,

Will this work, or should it be stringified?

Reply:
sorry for the double reviews :>

Reply:
pff is all good

Reply:
Im committing this during dota games xD

Reply:
should be just an array 🤔

Reply:
PSA I haven't ran this code either so im expecting it to blow up in some way 

Anybody said unit tests yes ?

Reply:
I think our code of conduct states that if we're not sure wether or not something works, we should push it ASAP so we can see if it works or not